sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/RSAKeyVault.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    /// <summary>
+    /// Provides an implementation of the RSA algorithm backed by Azure Key Vault.
+    /// </summary>
+    internal class RSAKeyVault : RSA
```

Comment:
Talking with @clairenovotny, it'd be nice to expose the public key as well.

Reply:
Talking with @schaabs it makes sense to return this class (make it public) so we don't later paint ourselves in a corner if we choose to add functionality like exposing the public key, which you could get from the certificate now anyway.